shell/edit/provisioning.cattle.io.cluster/__tests__/Advanced.test.ts

```diff
@@ -0,0 +1,533 @@
+/* eslint-disable jest/no-hooks */
+import { mount, Wrapper } from '@vue/test-utils';
+import Advanced from '@shell/edit/provisioning.cattle.io.cluster/tabs/Advanced.vue';
+import { _YAML, _VIEW, _EDIT } from '@shell/config/query-params';
+import { clone } from '@shell/utils/object';
+
+const PROV_CLUSTER = {
```

Comment:
can't all of these JS objects be moved to another file and exported to these tests? Personal opinion, but for me it makes the readability much easier. 

File would live in `provisioning.cattle.io.cluster/__tests__` as this one already does.

Reply:
Yes, they can be moved in `utils/file.ts`.